pages/room/index.js

```diff
@@ -18,8 +19,26 @@ const CreateRoom = () => {
     initSession();
   }, []);
 
-  function create() {
+  async function create() {
```

Comment:
can you change to arrow function just for consistency. 

`const create = async () => {...`

Reply:
Also, since it's an async function, we probably want a loading state so the user knows stuff is happening in the background before they're rerouted. You can do `setLoading(true)` at the beginning of this function and `setLoading(false)` at the end. Then in your return for this component, you can add a https://material-ui.com/components/progress/ somewhere